functions/lib/compression.service.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
```

Comment:
@jon-shipley I do not personally find much value in generating a service that only call library methods.
Are you planning on extending this any further?

Reply:
No.  The real value is that it is an adapter.  We can swap out the implementation at a later date.  We definitely do not want core business logic depending directly on external code, so we have this little module.